<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile_logged_page</class>
 <widget class="QWidget" name="profile_logged_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="welcome_lbl">
       <property name="text">
        <string>Welcome</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="email_lbl">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_count_lbl">
       <property name="text">
        <string>img count here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="like_count_lbl">
       <property name="text">
        <string>like count here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="verified_status_lbl">
       <property name="text">
        <string>verified status here</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="image_grid">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>520</width>
        <height>287</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="logout_btn">
     <property name="text">
      <string>Log out</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/assets/leave.png</normaloff>:/icons/assets/leave.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
